### ISSUE TITLE
John's running to do list

### ISSUE THREAD

**Issue by jmzobitz** [jmzobitz/BES-manuscript#23]
- [ ] Re-create model-diagram.pdf (I can't seem to find it on my local files. 😮‍💨 
- [ ] Resolve #19 and #9 (they are linked together)
- [ ] Finish revisions of plots to #17 
- [ ] Could we do a weekly gap fill instead of a monthly - how much better would it be
- [ ] flux-results: on the left text of each of the signifiers - fix dates - can you make it more rectangular? Is there a kind of label in plain English - thick horizontal line on the top row
- [ ] uncertainty-stats: density plot instead of a bar chart?
- [ ] Make table of reported RMSE, R2 for each of the sites (replace stats-plot.png)
 -[ ] Flux-results: adjust axes: Surface fluxes range from 0 5. Sub-level fluxes range from -10 - 20 (all three levels)  --> OR HOLD FIXED AT A SITE, but not between sites. X axis dates start at the first day, go in 6 hour increments, 3 hour minor grid.segments

**Comment by jmzobitz** [comment]
I am less inclined to see a density plot of the stats - here is one example:

![image](https://github.com/user-attachments/assets/68fcb4cd-da39-49f9-ba7b-47dcb0e4d49b)

**Comment by jmzobitz** [comment]
Signal to noise ratio
***********
John M. Zobitz, PhD
Pronouns: He, him, his

Professor of Mathematics and Data Science
Data Science Program Director
Environmental Studies Program
Augsburg University
2211 Riverside Avenue, CB 93
Minneapolis, MN 55454

Hagfors Center 187
P: (612) 330-1068

**Comment by jmzobitz** [comment]
@naupaka 

This would be helpful:

- [ ] keywords (up to 8)
- [ ] author order

**Comment by naupaka** [comment]
For Ed's figure 2 comment, see also https://github.com/jmzobitz/neonSoilFlux/issues/26

In a future release, may be worth considering filtering to remove sensor data from below lowest CO2 sensor depth.
